src/PrestaShopBundle/Resources/config/services/adapter/order.yml

```diff
@@ -166,3 +166,11 @@ services:
     tags:
       - name: tactician.handler
         command: PrestaShop\PrestaShop\Core\Domain\Order\Command\SendProcessOrderEmailCommand
+
+  prestashop.adapter.order.command_handler.cancel_order_product_handler:
+    class: PrestaShop\PrestaShop\Adapter\Order\CommandHandler\CancelOrderProductHandler
+    arguments:
+      - "@prestashop.core.localization.locale.context_locale"
```

Comment:
The arguments don't match the constructor

Reply:
done